habitat-lab/habitat/tasks/rearrange/multi_task/pddl_domain.py

```diff
@@ -226,15 +259,15 @@ def parse_predicate(
             ) from e
         return pred
 
-    def _parse_only_logical_expr(
+    def parse_only_logical_expr(
```

Comment:
Why make this method public? It looks like it is only used in this file anyways.

Reply:
This is an important API to expose to code outside of Habitat-Lab / Habitat-Baselines for setting up custom tasks.